zenoh/src/net/routing/dispatcher/pubsub.rs

```diff
@@ -143,7 +143,7 @@ pub(crate) fn undeclare_subscription(
                 }
             },
             None => {
-                tracing::error!(
+                tracing::trace!(
```

Comment:
This one should not occur with proper ACL (expo is not empty). Please revert to error.